### PR TITLE
Add orthomapping guides in 3 parts

### PR DESCRIPTION
**Pull request by CMPeng**
Addressing issue https://github.com/ArcGIS/geosaurus/issues/2268, this series of orthomapping guides have adopted Zikang and Rohit's sample notebooks, and been revamped into three parts -
1. setup and create image collection
2. block adjustments, control points, and create orthomosaic and DEM products
3. manage the handled images

### CODE REVIEW

**Comment by CMPeng** [comment]
@AtmaMani Please see the new commit with @Yongyao 's suggestions addressed, and I have also added a fly-over animation as gif towards the end of step 3. Thanks!

File: guide/06-imagery-and-raster-analysis/orthomapping_guide_1.ipynb [hunk 1]
```diff
@@ -0,0 +1,741 @@
+{
```
Comment (AtmaMani) [inline guide/06-imagery-and-raster-analysis/orthomapping_guide_1.ipynb]: <p>Can you rename variable names so they follow PEP8 coding standards? this applies for the entire notebook series.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/495/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (AtmaMani) [inline guide/06-imagery-and-raster-analysis/orthomapping_guide_1.ipynb]: <p>Add a line here saying you are defining a function that utilizes the <code>exifread</code> library to get location information out of the images. These coordinates would be utilized later in analysis.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/495/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Comment (AtmaMani) [inline guide/06-imagery-and-raster-analysis/orthomapping_guide_1.ipynb]: <p>Add a line saying the camera make in this example is <code>Yuneec</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/495/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

**Comment by AtmaMani** [comment]
@CMPeng this is a pretty good guide. I made some suggestions using reviewnb app, please use that to see the comments and let me know your thoughts. Part 3 looks good, I did not make any suggestions fro that.

**Comment by CMPeng** [comment]
Thanks @AtmaMani  for the remarks.
Can you please review again? The latest commits addressed the issues you have mentioned.

**Comment by AtmaMani** [comment]
@CMPeng everything looks good. I asked for one more change in the 2nd notebook "Getting Adjusted" segment.

**Comment by CMPeng** [comment]
Thanks @AtmaMani 
I have now removed the mentioning of "block adjustment window" phrase. Please check again :)

**Pull request merged**